<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-5738PF4F" failures="2" tests="8" name="FirefoxTest" time="0.037" errors="0" timestamp="2023-02-13T22:45:10 IST">
  <testcase classname="TestNG.PracticeClass_1" name="test1" time="0.002">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Home page url is not found did not expect [https://www.facebook.com/] but found [https://www.facebook.com/],
	Home page url is not found expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Home page url is not found did not expect [https://www.facebook.com/] but found [https://www.facebook.com/],
Home page url is not found expected [false] but found [true]
at TestNG.PracticeClass_1.test1(PracticeClass_1.java:67)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase classname="TestNG.PracticeClass_1" name="test2" time="0.001"/>
  <testcase classname="TestNG.PracticeClass_1" name="test3" time="0.001">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	forcefully fail this method">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
forcefully fail this method
at TestNG.PracticeClass_1.test3(PracticeClass_1.java:90)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="TestNG.PracticeClass_1" name="test4" time="0.001"/>
  <testcase classname="TestNG.PracticeClass_2" name="testA" time="0.001"/>
  <testcase classname="TestNG.PracticeClass_2" name="testB" time="0.003"/>
  <testcase classname="TestNG.PracticeClass_2" name="testC" time="0.0"/>
  <testcase classname="TestNG.PracticeClass_2" name="testD" time="0.0"/>
</testsuite> <!-- FirefoxTest -->
